wagtail/admin/templates/wagtailadmin/generic/chooser/chooser.html

```diff
@@ -7,10 +7,10 @@
 {% endif %}
 
 {% comment %} Do not attach JavaScript behaviour (data-tabs) below if there are no actual tabs used {% endcomment %}
-<div class="w-tabs" {% if creation_form %}data-tabs data-tabs-disable-url{% endif %}>
+<div class="w-tabs" data-controller="w-tabs">
```

Comment:
Be careful here, see the comment above, it explicitly states that we should not add tabs behaviour if there are not any tabs used.



```suggestion
<div class="w-tabs" {% if creation_form %}data-controller="w-tabs"{% endif %}>
```

Also, please update the comment itself to be something like.

```
{% comment %} Do not attach JavaScript behaviour (data-contoller="w-tabs") below if there are no actual tabs used {% endcomment %}
```